parm/config/config.base.emc.dyn

```diff
@@ -132,6 +132,8 @@ export COMINice=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ice
 export COMOUTice=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ice
 export COMINaero=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/chem
 export COMOUTaero=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/chem
+export COMINland=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos
+export COMOUTland=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos
```

Comment:
Not sure these are going to be used as the variables defined are from `ROTDIR` in the j-job.
May I know the motivation to add this here?

Reply:
@aerorahul The `COMOUTland` is defined here because it will be used later in the `ush/python/pygfs/task/land_analysis.py` following the aero_analysis.py. If you want me to remove `COMINland/COMOUTland`, I will replace `COMOUTland` with `COMOUTatmos` in the `ush/python/pygfs/task/land_analysis.py`.

Reply:
As suggested, removed the defined `COMINland` and `COMOUTland`.